lambdas/service/app.py

```diff
@@ -185,37 +239,56 @@ def openapi():
 
 
 health_spec = {
-    'responses': {
-        '200': {
-            'description': '200 response',
-            'content': {
-                'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
-                }
-            }
-        }
-    },
     'tags': ['Health']
 }
 
 
 @app.route('/health', methods=['GET'], cors=True, method_spec={
     **health_spec,
-    'summary': 'List health information for webservice'
+    'description': 'List health information for Azul service specific components.',
+    'summary': 'Check Azul service health',
+    'responses': {
+        '200': {
+            '$ref': '#/components/responses/OkHealth'
+        },
+        '503': {
+            '$ref': '#/components/responses/UnavailableHealth'
+        }
+    }
 })
 @app.route('/health/{keys}', methods=['GET'], cors=True, path_spec={
     'parameters': [
         {
             'name': 'keys',
             'in': 'path',
             'required': True,
-            'schema': {'type': 'string'}
+            'schema': {
+                'type': 'array',
+                'items': {
+                    'type': 'string',
+                    'enum': [
+                        'elasticsearch',
+                        'queues',
+                        'progress',
+                        'api_endpoints',
+                        'other_lambdas'
+                    ]
+                }
+            }
         }
     ],
 }, method_spec={
     **health_spec,
-    'summary': 'List health information for a specific key',
-    'description': 'List health information for a specific key, or all keys if `/health/` is used.'
+    'description': 'List health information for a specific key, or all keys if `/health/` is used.',
```

Comment:
```suggestion
    'description': 'List health information for a specific key
```
There is another endpoint available for `/health` with no components specified, so documenting that here seems redundant.

Reply:
@danielsotirhos I addressed your request but I left the last section. The path that ends with `/` is not registered and can not be used in the example, so there is a not that addresses its existence. Let me know if you disagree.

Reply:
Then perhaps `keys` should not be a required parameters?
![Screen Shot 2020-02-13 at 2 57 53 PM](https://user-images.githubusercontent.com/46407464/74486498-e9dbf980-4e71-11ea-9996-03806fc89f7a.png)
vs
![Screen Shot 2020-02-13 at 3 01 25 PM](https://user-images.githubusercontent.com/46407464/74486516-f2343480-4e71-11ea-9622-6f7d95e09b4a.png)